static/js/tools/timeline_util.ts

```diff
@@ -246,6 +245,29 @@ interface StatsVarInfo {
   title: string;
 }
 
+/*
+ * Get url params
+ */
```

Comment:
Could you add a clarifying comment for the return value here?

Reply:
Added comment